lib/ofx.rb

```diff
@@ -37,7 +37,7 @@ def method_missing(meth, *args) #:nodoc:
       end
     end
 
-    def respond_to?(meth) #:nodoc:
+    def respond_to?(meth, include_all = false) #:nodoc:

```

Comment:
why the extra param here?

Reply:
This is a super old gem that was forked like 9 years ago. In my updates for Ruby 2.7.1 I ran into this error:

```
warning: OfxParser::CreditAccount#respond_to?(:to_ary) uses the deprecated method signature, which takes one parameter
```

Basically, this method is overloading the `respond_to?` method, but it's not passing the correct params that's expected. This doesn't change any behavior, just updates the syntax for newer rubies.